spec/model/taxon_spec.rb

```diff
@@ -8,13 +8,42 @@
     before(:each) do
       example_schema_string = GovukContentSchemaTestHelpers::Examples.new.get('taxon', 'taxon_with_child_taxons')
       @example_schema = JSON.parse(example_schema_string)
+      @topic_taxon = Taxon.new("education")
     end
 
     it "should return a content item from the content store" do
```

Comment:
You're testing that you're getting the title here, so I would update the test name to reflect that.